pkg/reconciler/tenancy/bootstrap/bootstrap_reconcile.go

```diff
@@ -44,7 +44,9 @@ func (c *controller) reconcile(ctx context.Context, workspace *tenancyv1alpha1.C
 
 	// bootstrap resources
 	wsClusterName := logicalcluster.From(workspace).Join(workspace.Name)
-	klog.Infof("Bootstrapping resources for org workspace %s, logical cluster %s", workspace.Name, wsClusterName)
+	now := time.Now()
+	klog.Infof("Bootstrapping resources for %s", wsClusterName)
+	defer klog.Infof("Bootstrapping resources for %s done after %s", wsClusterName, time.Since(now))
```

Comment:
This has to be in a closure, `time.Since()` is resolved before `defer()` fires.

https://go.dev/play/p/Z7h4vP5wE00

Reply:
dropping from the PR. but you are right

Reply:
done